account_view_adjust/__init__.py

```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
```

Comment:
This declare is not longer needed in v12.
This file can be simply the following
```
from . import models

```